testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -2930,13 +2936,49 @@ QUnit.test("search should be execute after paste", function(assert) {
 
         var $input = $(toSelector(POPUP_CONTENT_CLASS) + " " + toSelector(TEXTEDITOR_INPUT_CLASS));
         $($input.val("o")).trigger("input");
-        clock.tick();
+        this.clock.tick();
         assert.equal($(".dx-list-item").length, 2, "filters execute on input event");
+    });
 
-    } finally {
-        clock.restore();
-        fx.off = originalFX;
-    }
+    [
+        { loadDelay: 200, indicateLoading: false },
+        { loadDelay: 1000, indicateLoading: true }
+    ].forEach(({ loadDelay, indicateLoading }) => {
+        QUnit.test(`search with loading delay = ${loadDelay} should ${indicateLoading ? "" : "not"} lead to the load panel being displayed`, function(assert) {
+            const clock = sinon.useFakeTimers();
```

Comment:
Looks like we already do this here:
https://github.com/DevExpress/DevExtreme/pull/10984/files#diff-922b6fcabe4f5ae828645b18174c6277R2919

Can we use `this.clock` instead?

Reply:
Thank you, I'll fix it